### PR TITLE
Feature/servicebox layout

### PR DESCRIPTION
**Pull request by alexanfl**
http://servicebox-layout.pr.dev.xal.no

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -185,9 +185,9 @@ img {
 
 .servicebox {
     display: grid;
-    grid-template-columns: repeat(2, 1fr);
-    grid-gap: 10px;
-    grid-auto-rows: minmax(80px, auto);
+    grid-template-columns: repeat(4, 10em);
```
Comment (eigilhs) [inline assets/css/main.css]: I think `1fr` works better here.

Comment (eigilhs) [reply assets/css/main.css]: To elaborate: It scales better on small screens.

Comment (alexanfl) [reply assets/css/main.css]: Make the suggestion please 😬

Comment (alexanfl) [reply assets/css/main.css]: @eigilhs

File: assets/css/main.css [hunk 2]
```diff
@@ -207,6 +207,11 @@ img {
     .mobile-footer {
         display: grid;
     }
+
+    .servicebox {
+        grid-template-columns: repeat(2, 10em);
```
Comment (eigilhs) [inline assets/css/main.css]:
```suggestion
        grid-template-columns: repeat(2, 1fr);
```

File: assets/css/main.css (hunk 1 above)
Comment (eigilhs) [reply assets/css/main.css]: @alexanfl

Comment (eigilhs) [reply assets/css/main.css]: You missed one @alexanfl

**Comment by alexanfl** [comment]
@eigilhs

**Pull request merged**